python/cugraph-dgl/examples/multi_trainer_MG_example/workflow.py

```diff
@@ -226,13 +231,15 @@ def run_workflow(rank, devices, scheduler_address):
     _ = load_dgl_dataset()
     del _
```

Comment:
I'm curious if this actually does anything since I thought freeing up memory is really up to the gc, and if you simply don't assign anything to the return value it's like a variable with a 0 ref count (ie. same outcome as calling `del` on a named var as done here)?  I could be wrong though, and I'm curious if you noticed something different to cause you to do this.

Reply:
This is a habit i have left from jupyter notebooks/ipython because they save variables in their OUT field. 

I think with Python we can safely remove it.

Reply:
Resolved with https://github.com/rapidsai/cugraph/pull/3603/commits/1cde19554778099d5ca62e71651753ca7b176dcd